packages/esm-patient-outcomes-app/src/smsform/send-sms-form.component.tsx

```diff
@@ -27,6 +36,7 @@ const SendSmsForm: React.FC<SendSmsFormProps> = ({
   const isTablet = useLayoutType() === 'tablet';
   const isOnline = useConnectivity();
   const { patientUuid, patient } = usePatient();
+  const { locale, allowedLocales } = useSession();
```

Comment:
I'm not entirely sure about this. What's the goal of locale switching here? Is the list of locales definitely the same as the list of locales for OpenMRS?